apps/dashboard/src/utils/slugify.ts

```diff
@@ -0,0 +1,9 @@
+export function slugify(text: string) {
```

Comment:
This can co into shared, to be reused between BE and FE

Reply:
@djabarovgeorge , could you please coordinate with @desiprisg to reuse the `slugify` utility you've added in a separate PR?

Reply:
sure, if we want to use signify function without dependencies so we can share it with the client we need to move this one to Shared package and delete [slugifyName](https://github.com/novuhq/novu/blob/add-env-switch-by-url/libs/application-generic/src/utils/slugify-name.ts#L3) from app-gen so everyone will use this one.

Reply:
I'll update the PR with this suggestion